src/functionalTest/resources/scenarios/RIA-8258-mark-appeal-as-remitted.json

```diff
@@ -0,0 +1,43 @@
+{
+  "description": "RIA-8258: Mark appeal as remitted",
+  "enabled": "{$featureFlag.homeOfficeGovNotifyEnabled}",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "AdminOfficer",
+    "input": {
+      "id": 8258,
+      "eventId": "markAppealAsRemitted",
+      "state": "ftpaDecided",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "isDlrmSetAsideEnabled": "Yes",
+          "listCaseHearingCentre": "taylorHouse",
```

Comment:
Shouldn't we use a feature flag for this new notification

Reply:
Need to set up a new LD flag, we don't have one yet for Remitted

Reply:
added new flag check in the FT